tin-drink/src/App.js

```diff
@@ -8,6 +8,10 @@ function xsh_djnui(){
   w = s + z;
 }
 
+function checkCOmmitId(){
+  let check = ' check';
```

Comment:
'check' is assigned a value but never used.
'check' is never reassigned. Use 'const' instead.